web-client/src/styles/custom.scss

```diff
@@ -1984,3 +1984,17 @@ button.change-scanner-button {
   background-size: 30% 30%;
   outline: 3px solid #005ea2;
 }
+
+.text-right {
+  text-align: right;
+}
+
+.width-0 {
```

Comment:
🔧 I believe USWDS also defines this as a class!

Reply:
ty! I removed this class; we are not even using it at this point.  I think we accidentally committed it.